src/Rules/Password.php

```diff
@@ -51,6 +51,8 @@ class Password implements Rule
      */
     public function passes($attribute, $value)
     {
+        $value = is_array($value) ? '' : (string) $value;
```

Comment:
Would it make more sense to flip this to an `is_scalar($value) ? (string) $value : ''` to also capture any possible scenario where the input is an object, although that wouldn't really happen with validating requests but could happen if someone were to run the Validator manually.

The current approach would also convert any class that has __toString implemented, not sure if you intended it like that?

Reply:
Good suggestion, done :thu